blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DAbstractMetricIO.java

```diff
@@ -183,7 +267,9 @@ public ResultSetFuture putAsync(Locator locator, long collectionTime, Rollup rol
      * @param range
      * @return
      */
-    protected <T extends Rollup> Table<Locator, Long, T> getRollupsForLocators(final List<Locator> locators, String columnFamily, Range range) {
+    protected <T extends Object> Table<Locator, Long, T> getRollupsForLocators(final List<Locator> locators,
```

Comment:
I was confused at first why this was changed from ```<T extends Rollup>``` to ```<T extends Object>```. But now I see you are using it to also fetch basic metrics from metrics_full. Should we change the name to something else since it no longer fetches just Rollup objects? Maybe ```getDataForLocators``` or ```getMetricsForLocators``` ?

Updating the javadoc/comment would also help :)